geoindicators/src/test/groovy/org/orbisgis/geoclimate/geoindicators/WorkflowGeoIndicatorsTest.groovy

```diff
@@ -197,6 +197,10 @@ class WorkflowGeoIndicatorsTest {
             // Check that the sum of proportion (or building floor area) for each RSU is equal to 1
             def utrfFloorArea = datasource."$geoIndicatorsCompute_i.rsu_utrf_floor_area"
             def colUtrfFloorArea = utrfFloorArea.getColumns()
+
+            // Test that the TYPO_SECOND is inside the RSU UTRF table
+            assertEquals colUtrfFloorArea.count("TYPO_SECOND"), 1
```

Comment:
revert to 

assertEquals 1, colUtrfFloorArea.count("TYPO_SECOND")

the argument order is expected, actual